apel/db/loader/loader.py

```diff
@@ -197,7 +197,10 @@ def load_msg(self, msg_text, signer):
                     apel.db.records.sync.SyncRecord: 'Sync',
                     apel.db.records.cloud.CloudRecord: 'Cloud',
                     apel.db.records.cloud_summary.CloudSummaryRecord:
-                    'Cloud Summary'}
+                    'Cloud Summary',
+                    apel.db.records.accelerator.AcceleratorRecord: 'Accelerator',
+                    apel.db.records.accelerator_summary.AcceleratorSummary: 'Accelerator Summary',
```

Comment:
Do we need trailing comma ?

Reply:
Yes, I generally prefer trailing commas in multi-line data structures as it means future additions don't change the last line.